extensions/binding/org.eclipse.smarthome.binding.dmx/README.md

```diff
@@ -0,0 +1,125 @@
+
+### Dimmer Thing (`dimmer`)
+
+The only mandatory configuration value for a dimmer thing is the `dmxid`, a list of DMX channels that are associated with this thing. There are several possible formats: `channel1,channel2,channel3,...` or `channel/width` or a combination of both. 
+
+The `fadetime` option allows a smooth fading from the current to the new value. The time unit is ms and the interval is for a fade from 0-100%. If the current value is 25% and the new value is 75% the time needed for this change is half of `fadetime`. Related is the `dimtime` option. It defines the time in from 0-100% if incremental dimmming is used. 
+
+Advanced options are the `turnonvalue`and the `turnoffvalue`. They default to 255 (equals 100%) and 0 (equals 0%) respectively. This value can be set individually for all DMX channels, the format is `value1,value2, ...` with values from 0 to 255. If less values than DMX channels are defined, the values will be re-used from the beginning (i.e. if two values are defined, value1 will be used for channel1, channel3, ... and value2 will be used for channel2, channel4, ...). These values will be used if the thing receives a ON or OFF command. 
+
+## Channels
+
+| Type-ID     | Thing        | Item          | Description                              |
+|-------------|--------------|---------------|------------------------------------------|
+|brightness   |dimmer        |Switch, Dimmer | controls the brightness of the channels  |
+|color        |dimmer        |Dimmer         | allows to set the color of the dimmer    |
+|control      |chaser        |String         | allows to change the chaser steps        |
+|switch       |dimmer,chaser |Switch         | turns the channel/chaser ON or OFF       |
+|mute         |(all bridges) |Switch         | mutes the DMX output of the bridge       |
+
+*Note:* the string send to the control channel of chaser things has to be formatted like the `steps` configuration of the chaser thing. If the new string is invalid, the old configuration will be used.
+
+## Full Example
+
+This example defines a sACN/E1.31 bridge in unicast mode which transmits universe 2 and three things: a three channel dimmer used to control a RGB light, a single channel dimmer which will turn on only to 90% if it receives a ON command and 
+
+### demo.things:
+
+```
+Bridge dmx:sacn-bridge:mybridge [ mode="unicast", address="192.168.0.60", universe=2] {
+ dimmer rgb    [ dmxid="5/3", fade=1000]
+ dimmer single [dmxid="50", fade=1000, turnonvalue="230" ]
+ chaser ampel  [dmxid="10,12,13", steps="100:255,0,0:1000|100:255,255,0:500|100:0,0,255:1000|100:0,255,0:500]"] 
+}
+```
+
+### demo.items:
+
+```
+Color MyColorItem "My Color Item" { channel="dmx:dimmer:rgb:color" }
+Dimmer MyDimmerItem "My Dimmer Item" { channel="dmx:dimmer:single:brightness" }
+Switch MyChaserItem "My Chaser Item" { channel="dmx:dimmer:chaser:ampel" }
+```
+
+### demo.sitemap:
+
+```
+t.b.a.
```

Comment:
Have a look at the [Hue binding](https://github.com/triller-telekom/smarthome/blame/master/extensions/binding/org.eclipse.smarthome.binding.hue/README.md#L119) for example